cvat/apps/engine/frame_provider.py

```diff
@@ -369,7 +369,7 @@ def iterate_frames(
         quality: FrameQuality = FrameQuality.ORIGINAL,
         out_type: FrameOutputType = FrameOutputType.BUFFER,
     ) -> Iterator[DataWithMeta[AnyFrame]]:
-        frame_range = itertools.count(start_frame, self._db_task.data.get_frame_step())
+        frame_range = itertools.count(start_frame)
```

Comment:
Probably, `db_segment_frame_set` below has to be converted to relative ids. There is a class method for this

Reply:
Ok, please call `dev/format_python_code.sh`

Reply:
Why not to add it to git pre-commit hook?

Reply:
There is a problem with determining the right interpreter in a cross-platform manner.